test/requests/completion_test.rb

```diff
@@ -210,10 +213,10 @@ class Baz
 
   def test_completion_for_constants
     source = +<<~RUBY
-      class Foo
+      class VSCode
```

Comment:
I use `VSCode` because `V` is the least conflicting name with Ruby's default class/modules.